packaging/pep517_backend/_backend.py

```diff
@@ -145,14 +151,58 @@ def build_sdist(  # noqa: WPS210, WPS430
             )
         )
 
+        tmp_sdist_directory = tmp_dir / "sdist"
+
         built_sdist_basename = _setuptools_build_sdist(
-            sdist_directory=sdist_directory,
+            sdist_directory=tmp_sdist_directory,
             config_settings=config_settings,
         )
 
+        built_sdist = Path(tmp_sdist_directory) / built_sdist_basename
+        output_sdist = Path(sdist_directory) / built_sdist_basename
+
+        source_date_epoch = int(os.environ.get("SOURCE_DATE_EPOCH", time.time()))
+
+        reproducible_sdist(built_sdist, output_sdist, source_date_epoch)
+
     return built_sdist_basename
 
 
+def reproducible_sdist(original_path: Path, output_path: Path, mtime: int) -> None:
```

Comment:
Function names should use verbs, from this name I can't guess what the side effects of the function are, for example.
Is it "make the incoming sdist reproducible"? "repack the sdist"? Something else?